Orange/widgets/visualize/owscatterplot.py

```diff
@@ -380,10 +380,16 @@ def update_graph(self, attributes=None, reset_view=True, **_):
         self.graph.zoomStack = []
         if attributes and len(attributes) == 2:
             self.attr_x, self.attr_y = attributes
+        X = BasicStats(self.data[:, self.attr_x], self.attr_x)
+        Y = BasicStats(self.data[:, self.attr_y], self.attr_y)
+        if X.nans or Y.nans:
+            self.warning("Some of the attributes contain missing values") 
+        else:
+             self.warning()      
         if not self.graph.have_data:
             return
         self.graph.update_data(self.attr_x, self.attr_y, reset_view)
-
+    
```

Comment:
The  spaces  are present as they were present before. I haven't changed any of that.
So, can you merge the PR or some more changes do yo expect?